python/tests/experimental/extras/test_configstore.py

```diff
@@ -0,0 +1,75 @@
+import os
+from tempfile import TemporaryDirectory
+
+from semantic_version import Version
+
+from whylogs.experimental.extras.configstore import LocalGitConfigStore
+
+
+def test_ctor():
+    with TemporaryDirectory(dir="/tmp") as prefix:
+        # create empty repo at prefix/my_org/my_model
+        cs = LocalGitConfigStore("my_org", "my_model", "test", repo_path=prefix)
+        assert not cs.exists()
+        cs.create()
+        assert cs.exists()
+        assert os.path.isdir(f"{prefix}/my_org/my_model")
+        assert os.path.isfile(f"{prefix}/my_org/my_model/test")
+        assert cs.get_available_versions() == [Version("0.0.0")]
+        assert cs.get_version_of_latest() == Version("0.0.0")
+        assert cs.get_latest() == ""
+
+        # repeat with existing directory
+        cs = LocalGitConfigStore("my_org", "my_model", "test", repo_path=prefix)
+        assert cs.exists()
```

Comment:
what happens if we call cs.create() on an existing config store? should we create a test for it?